frontend/ui/src/pages/Inbox/MessageInput/index.tsx

```diff
@@ -292,6 +338,12 @@ const MessageInput = (props: MessageInputProps & ConnectedProps<typeof connector
   };
 
   const selectSuggestedReply = (reply: SuggestedReply) => {
+    if (selectedSuggestedReply) setSelectedSuggestedReply(null);
+
+    if (input) setInput('');
+
+    if (selectedTemplate) setSelectedTemplate(null);
+
```

Comment:
here improved Ux: the newest selected templates or suggested reply is the one that gets selected in the input, overwriting any previously selected templates or suggested reply